### DISCUSSION TITLE
Is DT mod plays not calculating PP?

### DISCUSSION THREAD

**Discussion by xanning** [ppy/osu#25735] [Q&A] [answered]
I dont wanna switch stable. For me, its UI, gameplay everything is bad, and not new player friendly (i'm not)
Anyways, most of my plays with DT or NC doesnt seem like calculating PP
Is it a bug or?

**Comment by Joehuu** [comment, marked as answer]
If you have adjusted the rate, pp is currently not being calculated for those scores for now (duplicate of https://github.com/ppy/osu/discussions/25642).

**Reply by xanning** [reply]
How can i play the map again without increasing rate and get PP, because doesnt that mean i'm PP locked on that beatmap now?
Also, some maps that i didnt used DT been calculating for over a month too
![image](https://github.com/ppy/osu/assets/119341546/01f757e3-39bb-475f-9d98-d28733a52746) (no dt)
![image](https://github.com/ppy/osu/assets/119341546/9a532556-9dad-4ca7-a559-2914f643da81) (normal dt)
(same maps different diffs)
Another person that got this score but havent been calculated for 11 months
![image](https://github.com/ppy/osu/assets/119341546/90a04a9c-c970-4685-b25c-28a29cd50ebb)

**Reply by Joehuu** [reply]
> How can i play the map again without increasing rate and get PP, because doesnt that mean i'm PP locked on that beatmap now?

Rate adjustments not being calculated are temporary right now. Maybe it'll be enabled once scoring is finalized?

> Also, some maps that i didnt used DT been calculating for over a month too

Scores with new mods in lazer are also not calculated, see https://github.com/ppy/osu-queue-score-statistics/blob/eebd505535b1a2a8f8780b676acb941a29b5c06f/osu.Server.Queues.ScoreStatisticsProcessor/Processors/ScorePerformanceProcessor.cs#L212, pending consideration and balancing.

**Comment by xanning** [comment]
> Scores with new mods in lazer are also not calculated, see https://github.com/ppy/osu-queue-score-statistics/blob/eebd505535b1a2a8f8780b676acb941a29b5c06f/osu.Server.Queues.ScoreStatisticsProcessor/Processors/ScorePerformanceProcessor.cs#L212, pending consideration and balancing.


So i've been losing pp because ive been using Single tap..
Thanks for answering.
